rfcs/0146-meta-categories.md

```diff
@@ -266,9 +266,11 @@ Still unsolved is what data structure is better suited to represent a category.
 # Future work
 [future]: #future-work
 
-- Curate the categories.
-- Update documentation.
-- Update Continuous Integration.
+- Create a team to manage categorization-related issues, including but not
```

Comment:
I think this needs to be in the body as we are explicitly confirming the authority of the team to make some global-impact decisions (like when CI deployment is good enough for green-red instead of green-gray)?

Reply:
Done.